doc/getting_started/index.rst

```diff
@@ -42,9 +44,8 @@ procedure is currently supported on MacOS, Linux and Windows.
 
       .. code-block:: console
 
-
         # make sure you have the patch command available
-        cmake -G "NMake Makefiles" -S.. -DCMAKE_TOOLCHAIN_FILE=../xmos_cmake_toolchain/xs3a.cmake
+        cmake -G "Ninja" --toolchain  ../xmos_cmake_toolchain/xs3a.cmake -S..
```

Comment:
In other documentation, we have not included the `-S` option explicitly.  From the [cmake documentation](https://cmake.org/cmake/help/latest/manual/cmake.1.html), it looks like a path without an explicit preceeding `-S` behaves the same as one with this option.  I prefer that we settle on a consistent use of this option, although I do not have a strong opinion either way about which is preferred.

This comment applies to multiple files in this pull request.

Reply:
Yes I have never used the -S before - but I tested it and it worked fine so kept it in line with the rest of the documentation.